### ISSUE TITLE
Include VLAN interface and IP addresses in the network information

### ISSUE THREAD

**Issue by jennierose** [linode/manager#8618]
After connecting my Linode to a new VLAN at `10.0.0.1/24` and rebooting, I can see that the host has the correct IP address configured on `eth1:` 

```
$ ip addr
1: lo: ...
    ...
2: eth0: ...
    ...
3: eth1: ...
    link/ether xx:xx:xx:xx:xx:xx brd ff:ff:ff:ff:ff:ff
    inet 10.0.0.1/24 brd 10.0.0.255 scope global eth1
       valid_lft forever preferred_lft forever
```

I can see where that VLAN interface was configured under the "Configurations" tab, but `10.0.0.1` is not included among the IP Addresses in the "Network" tab. It shows me the public IPv4 and IPv6 addresses configured on `eth0`, plus a second link-local IPv6 address, but doesn't have any indication that there's a second network interface configured on the node.

Likewise, when I check the network information via the API:

```
curl -H "Authorization: Bearer $TOKEN" https://api.linode.com/v4/linode/instances/xxxxxxxx/ips | python3 -m json.tool
{
    "ipv4": {
        "public": [
            {
                "address": "(this is the only IP shown)",
                ...
            }
        ],
        "private": [],
        "shared": [],
        "reserved": []
    },
    ...
}
```

It has no information about the VLAN!

This is leading to a little bit of trouble trying to programmatically manage my hosts with the [Linode Ansible Collection](https://github.com/linode/ansible_linode) – I want to configure nodes to connect to one another using the VLAN, but it's not very straightforward trying to match a host to its `10.0.0.*` address with the information that is readily available through the API, (I can get the *configuration* for `eth1` with the `/configs` endpoint, but the address should really be included in the list of IPs in to begin with).

In short – are there any plans to include VLAN configs into the more general network info available in the API and web UI?

**Comment by Hik77** [comment]
I sure do appreciate the feedback.  Thank you so much for all your help.

Best Regards, Sherif Hikal

**Comment by ashtarcommunications** [comment]
Also trying to manage a dynamic Ansible inventory and need VLAN IP info accessible via the API.

@jennierose did you end up coming up with a solution you were happy with?
